packages/blade/src/components/Toast/Toast.stories.tsx

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { Title } from '@storybook/addon-docs';
+import type { StoryFn, Meta } from '@storybook/react';
+import React from 'react';
+import { useToast } from './useToast';
+import { Toast } from './Toast';
+import type { ToastProps } from './';
+import { ToastContainer } from './';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import { Box } from '~components/Box';
+import { Button } from '~components/Button';
+import { Heading, Text } from '~components/Typography';
+
+const Page = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      componentName="Toast"
+      componentDescription="Toast is a feedback element to display temporary short messages in the interface"
+      figmaURL="https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?type=design&node-id=75839-1125191&mode=design&t=SLxhqgKm27oCjSYV-4"
+    >
+      <Title>Usage</Title>
+      <Sandbox>
+        {`
+        import { ToastContainer, useToast } from '@razorpay/blade/components';
+
+        function App(): React.ReactElement {
+          const toast = useToast();
+
+          // Integrating Blade Toast in your App
+          // 1. Render the ToastContainer component at the root of your app
+          // 2. Utilize the methods exposed via useToast hook to show/dismiss toasts
+          return (
+            <Box>
+              <ToastContainer />
+              <Button 
+                onClick={() => {
+                  toast.show({ content: 'Payment successful', color: 'positive' })
+                }}
+              >
+                Show Toast
+              </Button>
+            </Box>
+          );
+        }
+        
+        export default App;        
+      `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Components/Toast',
+  component: Toast,
```

Comment:
The leading icon on promotional toast seems misaligned?
<img width="353" alt="Screenshot 2024-02-13 at 12 48 11 PM" src="https://github.com/razorpay/blade/assets/24487274/d48b4ee0-afd7-4255-a1e0-5d19ee0ee2b3">

Reply:
Yeah actually because we are using `<Text size="medium">` since the content supports React.ReactNode. 

I've changed it to use `<Text size="small">`